dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -374,63 +376,63 @@ private void updateInfo(Tool originalTool, Tool newTool) {
     @Timed
     @UnitOfWork
     @Path("/{containerId}/updateTagPaths")
-    @ApiOperation(value = "Change the workflow paths", authorizations = {
-        @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, notes = "Tag correspond to each row of the versions table listing all information for a docker repo tag", response = Tool.class)
+    @ApiOperation(value = "Change the tool paths.", authorizations = {
+        @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, notes = "Resets the descriptor paths and dockerfile path of all versions to match the default paths.", response = Tool.class)
```

Comment:
I think the note would be a bit more clear if emphasized that the paths aren't reset to the default of the tool in the database, rather they are set to the defaults found in the tool that is sent in as a parameter.